packages/office-ui-fabric-react/src/utilities/selectableOption/SelectableDroppableText.types.ts

```diff
@@ -86,6 +87,11 @@ export interface ISelectableDroppableTextProps<T> extends React.HTMLAttributes<T
    */
   calloutProps?: ICalloutProps;
 
+  /**
+   * Custom properties for ISelectableDroppableText's Panel used to render options on small devices.
+   */
+  panelProps?: IPanelProps;
+

```

Comment:
The dropdown component renders a callout for a normal sized device, and a panel for small devices. Inside calloutProps, there's a field for specifying focustrapprops (and inside that, we can set the field forcefocusinsidetrap to be false). By setting these fields, we can click "outside" (ie, past the first page of items) and not have the focus forced to the top of the dropdown list. We need the ability to set these same props for the mobile implementation, since when the dropdown is rendered on mobile, it's a panel.

Reply:
I understand that panelProps only apply to Dropdown component (when rendered in small devices), and `SelectableDroppableText.types.ts` factors out common props to both Dropdown and ComboBox component. However, because `SelectableDroppableText.types.ts` already contains calloutProps I though it was confusing to devs to find panelProps on a different file. Moreover, eventually ComboBox could benefit of the same behavior dropdown has today for small devices. Let me know your thoughts, thanks!